acoular/spectra.py

```diff
@@ -253,6 +256,9 @@ class PowerSpectra(BaseSpectra):
     #: (set from block_size and overlap).
     num_blocks = Property(desc='overall number of FFT blocks')
 
+    # Is set to a user value if the given SamplesGenerator streams non-finite data
```

Comment:
There is no need to adjust the number of blocks that the `PowerSpectra` object should average. Instead, the classes `MaskedTimeSamples`  and `MaskedTimeInOut` exists, which can be used to set the evaluation time interval. Thus, I suggest to revert this change.

Reply:
I did not know this! Reverted with any other changes in `spectra.py`: [f00b1b2](https://github.com/acoular/acoular/pull/285/commits/f00b1b2604ccbfce8e5564335e30c11bd937dede)